frontend/src/components/InlineTextEdit.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <v-edit-dialog      
+      :return-value.sync="textAreaValue"
+      lazy
+      large
+      @save="$listeners.save(item, editPath, textAreaValue)"
+      @cancel="$listeners.cancel"
+      @open="open"
+    > <span :style="{padding: '4px'}">{{ textArea }}</span>
+      <v-textarea
+        v-if="isShowTextArea == true"
+        slot="input"
+        ref="editTextArea"
+        v-model="textAreaValue"
+        label="Edit here..."
+        single-line
+        counter
+      ></v-textarea>
+    </v-edit-dialog>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+import { findObjectValue } from "@/utility";
+
+@Component({
+  name: "InlineTextEdit"
+})
+export default class InlineTextEdit extends Vue {
+  @Prop({ default: () => ({}) })
+  item!: any;
+  @Prop() value!: any;
+  @Prop() keyPath!: string;
+
+  isShowTextArea: boolean = false;
+  textAreaValue: any = "";
+
+  get editPath() {
+    //Object that is use in making of firebase path URL to save data in database.
+    let path: any = {};
+    path["keyPathId"] = this.item[".key"] ? this.item[".key"] : "";
+    path["keyPath"] = this.keyPath ? this.keyPath : "";
+    path["itemPath"] = this.value;
+    return path;
+  }
+
+  get textArea() {
+    return findObjectValue(this.item, this.value);
```

Comment:
ok I will try this with lodash.get and try to remove use of `findObjectValue` method.

Reply:
`this.value` is not nested. we are not able to use `{ text: "Status", value: "restoration.status" }` because it is already in use in `computedComponent` and we have to give the same name in table value 
`eg:
  computedComponent = {
        status: InlineStatusEdit
  };
Then we have to use { text: "Status", value: "status" }. `

Reply:
`{ text: "Status", value: "restoration.status" }` not work with ` computedComponent = { status: InlineStatusEdit }`

Reply:
You can give computedComponent nested property to match the header value.

Reply:
computedComponent can have the property like this `{ "restoration.status": InlineStatusEdit }`

Reply:
thank you i understand it now.